Gems/Atom/RPI/Code/Source/RPI.Public/Pass/AttachmentReadback.cpp

```diff
@@ -580,7 +580,7 @@ namespace AZ
                 auto bufferSize = readbackBufferCurrent->GetBufferSize();
                 readbackItem.m_dataBuffer = AZStd::make_shared<AZStd::vector<uint8_t>>();
 
-                void* buf = readbackBufferCurrent->Map(bufferSize, 0);
+                void* buf = readbackBufferCurrent->Map(bufferSize, 0)[0]; //? Take element 0 here
```

Comment:
instead of 0 lets use an enum/cosnt value for the default device index with a comment explaining that we only support attachment read back on the default device for now